loading-kernel-module/machineconfig.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
+  name: 99-ybettan-external-image
+spec:
+  osImageURL: quay.io/ybettan/coreos-layering:simple-kmod
```

Comment:
This won't actually work in OCP because we require digested pull specs today.  Also, it's OCP specific so we can probably drop it.